src/main/java/website/frontrow/board/Bubble.java

```diff
@@ -15,9 +17,23 @@
 {
     public static final SpriteStore SS = new SpriteStore();
 
+    private static final Point HIT_MOTION = new Point(0, -2);
```

Comment:
I think the variable name HIT_MOTION is a bit unclear, maybe change it to something like FLOAT_UP_MOTION.